snuba/datasets/errors_replacer.py

```diff
@@ -396,17 +392,19 @@ def process_unmerge(
 def process_delete_tag(
     message: Mapping[str, Any],
     schema: TableSchema,
-    tag_column_map: Mapping[str, Mapping[str, str]],
-    promoted_tags: Mapping[str, Sequence[str]],
+    promoted_column_spec: Mapping[str, PromotedColumnSpec],
```

Comment:
`promoted_columns_spec` with the s. Should we have a separate class for this if it always only contains "tags" and "contexts"?